lib/src/tinycolor.dart

```diff
@@ -161,4 +161,16 @@ class TinyColor {
   Color get color {
     return _color;
   }
+
+  @override
+  bool operator ==(Object other) =>
+    identical(this, other) ||
+    other is TinyColor &&
+    runtimeType == other.runtimeType &&
+    name == other.name;
+
+  @override
+  int get hashCode => name.hashCode;
+
+  bool equals(Object other) => this == other;
```

Comment:
Why do we need `equals()` if we have `==`?

Reply:
The original JS version use `.equals()`, so I just make an alias method for JS developers. Maybe I will add `@Deprecated` at `.euqals()` to remind them to use `==` instead?